app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V1
+    class MessagesController < ApiController
+      def create
+        message = MessageService.new(message_params, current_user, params[:conversation_id]).create
```

Comment:
define a safe parameter for conversation_id, instead of using direcly params[:conversation_id], as you did for the message_params